integration-tests/resources/jobs/tests/kafka_to_redis.yaml

```diff
@@ -0,0 +1,15 @@
+input:
+  uses: kafka.read
+  with:
+    bootstrap_servers: kafka
+    topic: "integration-tests"
+    group: "integration-tests"
```

Comment:
You can omit double quotes (should be added normally when having a number, but still it should be a string).